tests/fsharp/Compiler/Language/InterfaceTests.fs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+
+[<TestFixture>]
+module InterfaceTests =
+
+    [<Test>]
+    let ShouldWork() =
+        CompilerAssert.Pass
+            """
+type IGet<'T> =
+    abstract member Get : unit -> 'T
+    
+type GetTuple() =
+    interface IGet<int * int> with
```

Comment:
These pass because `optIntfSlotTys` is empty and doesn't go through the interface naming.